### PR TITLE
Bump React to 16.14

### PR DESCRIPTION
**Pull request by eatyourgreens**
[React 16.14](https://github.com/facebook/react/blob/main/CHANGELOG.md#16140-october-14-2020) was released alongside React 17 (October 2020) and backports the [new JSX transform](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html) to React 16. I've updated `app-content-pages` by hand, and run the [`update-react-imports` codemod](https://github.com/reactjs/react-codemod/blob/master/README.md#update-react-imports) across `app-project/pages` and `app-project/src`.

TLDR: `import React from 'react'` is no longer required to render JSX, and can be replaced by destructured imports for Hooks etc. If a component does need `React`, use `import * as React from 'react'`.

Package:
app-content-pages
app-project
lib-classifier
lib-react-components

# Review Checklist

## General

- [x] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

**Review by shaunanoordin — COMMENTED** [review]
## PR Review

This is a dependency update, but a semi-sorta-significant one since it bumps React by a minor version yet touches 310 files(!)

Code read itself looks good - we're just removing `import React from 'react'` (or replacing it with `import * as React from 'react' on two or three places)` so no functional changes are expected.

Let me run some tests (mostly app-project + I Fancy Cats) to ensure nothing big was missed.

(Also, I'm assuming this takes precedence over #2281 which attempts to have a major update to React 17)

### Status

Safety testing in progress

File: packages/app-project/pages/_app.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { Box } from 'grommet'
 import makeInspectable from 'mobx-devtools-mst'
 import { Provider } from 'mobx-react'
 import Error from 'next/error'
-import React, { useEffect, useMemo } from 'react'
+import { useEffect, useMemo } from 'react';
```
Comment (shaunanoordin) [inline packages/app-project/pages/_app.js:5]: [minor][code style] During the refactor, a bunch of semicolons `;` have been added by accident. I think there are about 22 instances of this.

**Review by shaunanoordin — APPROVED** [review]
## PR Review (Update)

Safety tests look fine 👍 

I made a minor note about code styling; in a few places, semicolons `;` were accidentally introduced. I didn't mark all 22-ish instances, but they should be easily found by searching for the text "`'react';`"

### Testing

Tested on localhost with macOS 10.12.6 + Chrome 91. Primarily tested `app-project` (with `yarn build && yarn start`).

**Scenario 1:** I Fancy Cats
- Our standard project. URL: http://localhost:3000/projects/brooke/i-fancy-cats/classify/workflow/693?env=staging
- I can open the home page, I can navigate to the classify page
- I can view the Subject image and pan/zoom/etc (Single Image Viewer)
- I can make annotations and edit them (Single Question Answer, Multi Answer Question, Drawing Task (Point, Ellipse))
- I can submit Classifications

**Scenario 2:** TESS
- Our flagship FEM project. (No active subjects at the moment, tho.) URL: http://localhost:3000/projects/nora-dot-eisner/planet-hunters-tess?env=production
- I can view the Subject data and pan/zoom/etc (Light Curve Viewer)
- I can make annotations and edit them (2D range markers)
- I can submit Classifications

### Status

LGTM 👍

File: packages/app-project/pages/_app.js (hunk 1 above)
Comment (eatyourgreens) [reply packages/app-project/pages/_app.js:5]: Oh, I think the React codemod must have done that.

**Pull request merged**